resources/assets/js/views/reports/ExpensesReport.vue

```diff
@@ -92,19 +92,19 @@ export default {
     return {
       range: new Date(),
       dateRange: [
-        'Today',
-        'This Week',
-        'This Month',
-        'This Quarter',
-        'This Year',
-        'Previous Week',
-        'Previous Month',
-        'Previous Quarter',
-        'Previous Year',
-        'Custom',
+        'Dnes',
```

Comment:
@BIIINO Hey, changing this list directly will update it for all languages (it'll show date range fields in Slovak even if you select english language)

In order to make this date range fields translatable, we need to add all of these strings as keys on `en.json` etc.